CrackingTheCodingInterview/Chapter7MathematicsAndProbability/Operations.java

```diff
@@ -40,6 +40,9 @@ public static int abs(int a) {
 		else {
 			return a;
 		}
+		
+		//or
+		return Math.abs(a);
```

Comment:
Thanks for the PR. 

However, I don't think you understood the concept of this problem. You're not allowed to use anything else besides the add operation, meaning you're forbidden to use Math.abs for two reasons. Also, I'm pretty certain Math.abs uses multiplication internally. 

Two other less important reasons:
1) Your code is unreachable -- see if you can convince yourself of this.
2) Even if your code was reachable, "a" will already be in absolute value form so calling Math.abs here is a no op. 

Because of these reasons, I'm closing out the PR.